product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -1,12 +1,19 @@
 ---
 title: "Setting up your identity provider"
 navTitle: "Setting up your identity provider"
-description: Applies when you purchased BigAnimal directly from EDB
+description: How to set up your identify provider when you purchased BigAnimal directly from EDB
 ---
 
-If you purchased BigAnimal directly from EDB, you must set up your identity provider (IDP) before accessing BigAnimal for the first time. After setting up your identity provider, you can add users to BigAnimal by adding them to the designated group in your identity provider. Once you've logged into BigAnimal using your identity provider, you can set up your cloud service provider (CSP) in the BigAnimal portal to complete onboarding. 
+If you purchased BigAnimal through EDB, you can sign in with your EDB account for the first time and set up your own identity provider if you wish. 
```

Comment:
Going to make that distinction by moving the note about Azure marketplace underneath the opening paragraph like so: 

```
After purchasing BigAnimal, you can sign in with your EDB account for the first time and set up your own identity provider if you wish. 

!!! Note
    If you purchased through Azure Marketplace, BigAnimal authenticates users using Azure Active Directory (AD) and you don't have to complete these steps. Azure AD is linked during subscription. Also, you can still invite users that have an EDB account through the BigAnimal portal. 
...
```
@xugy99 @drothery-edb Does that work?

Reply:
Hi @ccestes ,

> After purchasing BigAnimal,  you can .......

it is not very accurate from my understanding, because "any user with an EDB account, who signs in to BigAnimal, can always set up an identity provider, without pay to EDB for anything". 

So there is no "purchase" prerequisite for this statement here.